tf_encrypted/keras/testing_utils.py

```diff
@@ -26,6 +26,13 @@ def agreement_test(tfe_layer_cls, kwargs=None, input_shape=None,
   tf_layer_cls = getattr(tf.keras.layers, tfe_layer_cls.__name__)
   tfe_kwargs = {**kwargs, **tfe_kwargs}
 
+  # If the layer as a `fused` argument (e.g. BatchNormalization),
+  # the TF layer should be executed with fused=False. The main reason is
+  # that TF supports channel first only when fused=False
```

Comment:
I don't think this should matter since our implementation can handle both, right?

Reply:
I was getting a pb because `default_args_check(fused,..,..)` was expecting to have `fused=None` but it had to be `fused=False`  when running the test, otherwise TensorFlow couldn't run with `fused=None` and channel_first. 

But I am going to remove this meaningless test `default_args_check(fused,..,..)` so won't need to do this check.